src/Formatter/Simple.php

```diff
@@ -63,7 +63,7 @@ public function format($event)
 
         $event = parent::format($event);
         foreach ($event as $name => $value) {
-            if ('extra' == $name && count($value)) {
+            if ('extra' == $name && $value) {
```

Comment:
Not sure about this change, what if `$value` is `countable` object?

Reply:
ok, reverted the change

Reply:
In that case I would add test on it